src/util/client-info.ts

```diff
@@ -0,0 +1,42 @@
+import assign from "../util/assign";
+import "unfetch/polyfill";
```

Comment:
As per previous comment, no need for polyfills.

Reply:
Not very impressed with the `jest-mock-fetch` package. You can't specify "if something hits this URL, respond with this". You can only specify: "The first time someone makes a fetch call, respond with this, the second time a fetch call is made, respond with that." which is inherently brittle.